src/Acsint.jl

```diff
@@ -416,7 +403,7 @@ end
     δ = ⍵^2 / (⍵^2 + α)
     TT = T * δ
 
-    Fnmax = if T == 0.0 
+    Fnmax = if T < 1e-20    # T ≈ 0
```

Comment:
Can T be negative? If so this needs to be `abs`

Reply:
T should always be positive because its defined as `T = p * RPC ^ 2` where `p` is a sum of basis function exponents